tests/unit/test_state.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+#
+# Learn more about testing at: https://juju.is/docs/sdk/testing
+
+# pylint: disable=protected-access
```

Comment:
I don't think you need this `disable=protected-access` in this file

Reply:
thanks, this was left over from copy and paste.